src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -74,9 +74,9 @@ def show_storage_account_connection_string(cmd, resource_group_name, account_nam
     return {'connectionString': connection_string}
 
 
-def show_storage_account_usage(cmd):
+def show_storage_account_usage(cmd, location):
```

Comment:
Does this mean the `location` becomes a mandatory argument, and we are looking at a breaking change? If yes, then you really need to update the history.rst to tell people so. Also anyway we can still keep it optional?

Reply:
Yes, was just about to update the history.
According to storage team, the global list-usage now gives back no useful information. (if you do, you'll see a response telling you have zero storage accounts). Given this, I think it is best to make the breaking change and bump the minor version.